lib/Backend/BailOut.cpp

```diff
@@ -2161,6 +2181,21 @@ void BailOutRecord::ScheduleFunctionCodeGen(Js::ScriptFunction * function, Js::S
         rejitReason = RejitReason::Forced;
     }
 
+    // REVIEW: Temporary fix for RS1.  Disable Rejiting if it looks like it is not fixing the problem.
+    //         For RS2, turn this into an assert and let's fix all these issues.
+    if (!reThunk && rejitReason != RejitReason::None)
+    {
+        if (executeFunction->GetDynamicProfileInfo()->GetRejitCount() >= 100)
+        {
+            reThunk = true;
```

Comment:
rejitReason = RejitReason::None; will take care of not rejitting. 

---
In reply to: [65753177](https://github.com/Microsoft/ChakraCore/pull/1082#discussion_r65753177) [](ancestors = 65753177)